benchmark/coverage_analyzer.py

```
@@ -0,0 +1,1134 @@
+        
+        for node in ast.walk(tree):
+            for operator in self.mutation_operators:
+                try:
+                    node_mutations = operator(node, file_path)
+                    mutations.extend(node_mutations)
+                except Exception:
+                    continue
+        
+        return mutations
+    
+    def _mutate_arithmetic_operators(self, node: ast.AST, file_path: str) -> List[Dict[str, Any]]:
```

Ruff (0.12.2)

Unused method argument: `file_path`

(ARG002)